pkg/solana/relay.go

```diff
@@ -200,9 +201,30 @@ var _ relaytypes.MedianProvider = &medianProvider{}
 
 type medianProvider struct {
 	*configProvider
-	reportCodec median.ReportCodec
-	contract    median.MedianContract
-	transmitter types.ContractTransmitter
+	transmissionsCache *TransmissionsCache
+	reportCodec        median.ReportCodec
+	contract           median.MedianContract
+	transmitter        types.ContractTransmitter
+}
+
+// start both cache services
+func (p *medianProvider) Start(ctx context.Context) error {
```

Comment:
that was probably the biggest piece, but there was some slight mixing of `transmissionSigner` vs `transmissionsID` which are probably too closely named 😆 

but I am curious why the e2e tests in core didn't fail since it didn't work locally when I tried 🤔